rca/project_styleguide/templates/patterns/molecules/card/card.html

```diff
@@ -16,9 +16,18 @@
                     <div class="card__meta">{{ item.meta }}</div>
                 {% endif %}
                 <h3 class="card__heading">
-                    {% if item.link %}<span class="card__heading-link">{% endif %}
-                        {{ item.title }}
-                    {% if item.link %}</span>{% endif %}
+                    {% if item.link %}
+                         <span class="card__heading-link">
+                             {{ item.title }}
+                             {% if link_style  == "with-arrow" %}
+                             <svg class="card__heading-link-icon" width="12" height="8">
```

Comment:
I don't think this would be announced by most screen readers but it could be worth adding `aria-hidden="true"` for the ones that will (I think Narrator on Windows might).